.github/workflows/release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           pip list
           conda list
       - name: Build jar
-        run: python setup.py java
+        run: python setup.py build_ext
```

Comment:
This could be fused with the step below

Reply:
Good catch! Yeah I wanted to go through and consolidate all these instances where we're installing the Java twice